tests/unit/options/unit_tests_sender_option_parser.rb

```diff
@@ -358,6 +358,17 @@ def test_sender_option_parser_user_msg_content_from_file_long
     )
   end # test_sender_option_parser_user_msg_content_from_file_long
 
+  def test_sender_option_parser_user_content_type_msg_content_values
+    for type, param, expect in [%w(string 1 1), ["int", "1", 1],
+                               ["float", "1.1", 1.1], ["bool", "true", true]]
+      assert_equal(
+          expect,
+          Options::SenderOptionParser.new(
+              ["--content-type", type, "--msg-content", param]).options.msg_content
+      )
+    end
+  end # test_sender_option_parser_user_content_type_msg_content_list_item_values
+
```

Comment:
Should contain test for exception when --content-type is not in %w(string, int, long, float, bool).

And test of raised message, see other tests which ends "_raise", "_raise_message".

Reply:
done

Reply:
ok